README.md

```diff
@@ -1 +1,192 @@
-# capstone-2
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [toDoList] <a name="about-project"></a>
+
+**[toDoList]** is a website that allows you to check pokemons and comment abou it!.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML, CSS and JavaScript
+- Git, Github and Visual Studio Code
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[show_pokemons]**
+- **[select_pokemon]**
+- **[consume_api]**
+- **[get_api]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[You can see our live demo](https://github.com/Swarzstein/capstone-2/dist)
```

Comment:
- Optional. The deployed link is not working. Since deploying this project was not a part of the project requirements, you can consider removing this section. However, I would strongly suggest you deploy this project online so you can put it on your portfolio.